src/dex-ui/store/poolsSlice/utils.ts

```diff
@@ -1,25 +1,26 @@
 import { BigNumber } from "bignumber.js";
 import {
-  calculatePercentOfPool,
   calculateTotalValueLockedForPool,
   calculateUserPoolLiquidity,
   calculateVolume,
+  calculatePercentOfPoolFromTotalSupply,
 } from "../../utils";
-import { Pool, UserPool } from "./types";
-import { A_B_PAIR_TOKEN_ID, MirrorNodeTokenBalance, MirrorNodeTransaction, TokenPair } from "../../services";
+import { Pool, UserPool, TokenPair } from "./types";
+import { MirrorNodeTokenBalance, MirrorNodeTransaction } from "../../services";
 
 /**
  * TODO: This is mocked data that adds a token pair balance to the primary pool balance data.
  * This should be removed after we can fetch pair tokens from the pool contract.
  * */
-const appendLiquidityTokenBalance = (poolTokenBalances: MirrorNodeTokenBalance[], mockLPbalance: BigNumber) => {
+/* const appendLiquidityTokenBalance = (poolTokenBalances: MirrorNodeTokenBalance[], mockLPbalance: BigNumber) => {
```

Comment:
Let's remove instead of commenting out this block if this is no longer needed.

Reply:
Done